### PR TITLE
chore: mondaynet integration tests update mondaynet constants

### PR DESCRIPTION
**Pull request by zainen**
Thank you for your contribution to Taquito.

Before submitting this PR, please make sure:

- [ ] Your code builds cleanly without any errors or warnings
- [ ] You have run the linter against the changes
- [ ] You have added unit tests (if relevant/appropriate)
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

In this PR, please also make sure: 

- [ ] You have linked this PR to the issue by putting `closes #TICKETNUMBER` in the description box (when applicable)
- [ ] You have added a concise description on your changes
## Release Note Draft Snippet

__If relevant, please write a summary of your change that will be suitable for
inclusion in the Release Notes for the next Taquito release.__

### CODE REVIEW

**Review by hui-an-yang — COMMENTED** [review]
mondaynet ci failed, can you we re-run failed job to know it's working?

File: integration-tests/known-contracts-ProtoALph.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-export const knownContractProtoALph = "KT1Mkg3dfZL1yTUdYoFCmZ7CQVSpydcMjTR2";
-export const knownBigMapContractProtoALph = "KT18uDJMsdqxU2ox4dvLAn1XSsXK45a9V4Nd";
-export const knownTzip12BigMapOffChainContractProtoALph = "KT1KyeMPwLC7ExVnMr1wQGzCBKukwpDtZ1KJ";
-export const knownSaplingContractProtoALph = "KT1HiTa8uiGDey2DTodDHijTpcjJd62Yx5Rs";
-export const knownOnChainViewContractAddressProtoALph = "KT1TeHWECXnT1cDgigPk1f12jvYBEPR5TXiF";
+export const knownContractProtoALph = "KT1W8LnB5YMYDnNk1Z9BpMkZnWEEWCvZgP9q";
+export const knownBigMapContractProtoALph = "KT1TJSCWVhSKf1vFYHd1DjaMsWYc3YEknCst";
+export const knownTzip12BigMapOffChainContractProtoALph = "KT1KFU8hN2xJoVufPxyeKHLCf5TwELAEquS2";
+export const knownSaplingContractProtoALph = "KT1HpMhQF8jtvC4gYdTTZ1Hatou4i7aXv32X";
+export const knownOnChainViewContractAddressProtoALph = "KT1FsXv8CXL8ALZEFXgKQ4JFLWmH2WdLR7Vu";
```
Comment (AlirezaHaghshenas) [inline integration-tests/known-contracts-ProtoALph.ts:5]: I think these values are generated each time we run integration tests. If so, is it better to exclude them from commits?

Comment (zainen) [reply integration-tests/known-contracts-ProtoALph.ts:5]: only changed if we do `npm run originate-known-contracts-and-run-tests` and unsure if every monday the contracts are automatically updated

**Comment by zainen** [comment]
will check why the contracts are there but are seen as missing in the CI

**Review by hui-an-yang — APPROVED** [review]
lgtm!

**Review by dsawali — APPROVED** [review]

File: integration-tests/known-contracts-ProtoALph.ts (hunk 1 above)
Comment (dsawali) [reply integration-tests/known-contracts-ProtoALph.ts]: i think we might not need to commit them weekly though, they should also be created during CI I believe. Which is what Ali might be implying

**Pull request merged**